agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/LogDataMapper.java

```diff
@@ -100,11 +102,12 @@ public LogDataMapper(
     this.telemetryInitializer = telemetryInitializer;
   }
 
-  public TelemetryItem map(LogRecordData log, @Nullable String stack, @Nullable Long itemCount) {
+  public TelemetryItem map(LogRecordData log, @Nullable Long itemCount) {
+    String stack = log.getAttributes().get(SemanticAttributes.EXCEPTION_STACKTRACE);
     if (stack == null) {
       return createMessageTelemetryItem(log, itemCount);
     } else {
-      return createExceptionTelemetryItem(log, itemCount, stack);
+      return createExceptionTelemetryItem(log, itemCount);
     }
   }
```

Comment:
`stack` was intentionally passed in as arg so we don't need to look it up multiple times